tests/foreman/ui/test_contentview.py

```diff
@@ -19,3853 +16,18 @@
 
 :Upstream: No
 """
-import datetime
-from random import randint
-
 import pytest
-from airgun.exceptions import InvalidElementStateException
```

Comment:
To avoid the larger diff, for rename you could simply run
`git mv tests/foreman/ui/test_contentview.py tests/foreman/ui/test_contentview_old.py`
and then create a new `tests/foreman/ui/test_contentview.py`

Reply:
@sambible Can you try this for rename?